package.json

```diff
@@ -192,7 +192,7 @@
                 "csharpextensions.includeNamespaces": {
                     "type": "boolean",
                     "default": true,
-                    "description": "If created classes should include default namespaces"
+                    "description": "If created classes should include default namespaces/usings"
```

Comment:
Could you remove the 'namespaces' part? It isn't descriptive and, as you found out, simply confusing.

Reply:
Sure, done!